doc/how-to-write-tests.txt

```diff
@@ -163,11 +163,11 @@ can instead embedd one of the fake home suites:
 * FakeHomeSuite: creates a fake home directory with ~/.ssh and fake ssh keys.
 * FakeJujuHomeSuite: as above but also sets up a ~/.config/juju with a fake model.
 
-The JujuConnSuite does this and more. It also sets up a controller and api
-server.  This is one problem with the JujuConnSuite, it almost always does a
+The ApiServerSuite does this and more. It also sets up a controller and api
```

Comment:
Is this all true still?

Reply:
Arguably it is. It's far less bad than it was. I think once we get more new infrastructure in place, we can revisit this text.